<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sysadm-Latitude-3480" ignored="0" name="Payment UI Validation" tests="8" failures="3" timestamp="05 Mar 2020 05:12:31 GMT" time="5.965" errors="0">
  <testcase name="@BeforeClass setUp" time="0.023" classname="paymentsUI.PaymentUI_SuccessfulPaymentMessage">
    <failure type="java.net.ConnectException" message="Connection refused (Connection refused)">
      <![CDATA[java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:607)
at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:121)
at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:180)
at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:326)
at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:610)
at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:445)
at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:835)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at org.apache.http.client.HttpClient$execute$2.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl$RestAssuredHttpBuilder.doRequest(RequestSpecificationImpl.groovy:2028)
at io.restassured.internal.http.HTTPBuilder.post(HTTPBuilder.java:349)
at io.restassured.internal.http.HTTPBuilder$post$8.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.sendRequest(RequestSpecificationImpl.groovy:1202)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1210)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1019)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:149)
at io.restassured.internal.filter.SendRequestFilter.filter(SendRequestFilter.groovy:30)
at io.restassured.filter.Filter$filter$2.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.time.TimingFilter.filter(TimingFilter.java:56)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.log.RequestLoggingFilter.filter(RequestLoggingFilter.java:124)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at io.restassured.filter.Filter$filter$2.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.FilterContext$next$1.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1631)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1210)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1019)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1637)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1210)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1019)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy:170)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy)
at domains.PaymentNodeJS.payUIget(PaymentNodeJS.java:1457)
at paymentsUI.PaymentUI_SuccessfulPaymentMessage.setUp(PaymentUI_SuccessfulPaymentMessage.java:28)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="paymentsUI.PaymentUI_SuccessfulPaymentMessage">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="@BeforeClass setUp" time="1.782" classname="paymentsUI.Payment_NodeJS_NB_CitiBank">
    <failure type="org.openqa.selenium.SessionNotCreatedException" message="session not created: This version of ChromeDriver only supports Chrome version 77
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;sysadm-Latitude-3480&amp;apos;, ip: &amp;apos;127.0.1.1&amp;apos;, os.name: &amp;apos;Linux&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;5.3.0-40-generic&amp;apos;, java.version: &amp;apos;1.8.0_242&amp;apos;
Driver info: driver.version: ChromeDriver
remote stacktrace: #0 0x558fec8b9959 &amp;lt;unknown&amp;gt;
">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: session not created: This version of ChromeDriver only supports Chrome version 77
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'sysadm-Latitude-3480', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '5.3.0-40-generic', java.version: '1.8.0_242'
Driver info: driver.version: ChromeDriver
remote stacktrace: #0 0x558fec8b9959 <unknown>

at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499)
at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:531)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:147)
at common.WrapperMethod.getDriver(WrapperMethod.java:263)
at paymentsUI.Payment_NodeJS_NB_CitiBank.setUp(Payment_NodeJS_NB_CitiBank.java:32)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="paymentsUI.Payment_NodeJS_NB_CitiBank">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="@AfterMethod afterMethod" time="0.03" classname="paymentsUI.PaymentUI_AddStoredCardAndValidate">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.remote.Augmenter.extractRemoteWebDriver(Augmenter.java:72)
at org.openqa.selenium.remote.BaseAugmenter.augment(BaseAugmenter.java:96)
at common.CommonUtil.screenshot(CommonUtil.java:179)
at common.CommonUtil.afterMethod(CommonUtil.java:142)
at paymentsUI.PaymentUI_AddStoredCardAndValidate.afterMethod(PaymentUI_AddStoredCardAndValidate.java:96)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="paymentsUI.PaymentUI_AddStoredCardAndValidate">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="@AfterMethod afterMethod" time="0.001" classname="paymentsUI.PaymentUI_ErrorValidationUPIWalletAndNB">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.remote.Augmenter.extractRemoteWebDriver(Augmenter.java:72)
at org.openqa.selenium.remote.BaseAugmenter.augment(BaseAugmenter.java:96)
at common.CommonUtil.screenshot(CommonUtil.java:179)
at common.CommonUtil.afterMethod(CommonUtil.java:142)
at paymentsUI.PaymentUI_ErrorValidationUPIWalletAndNB.afterMethod(PaymentUI_ErrorValidationUPIWalletAndNB.java:56)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="paymentsUI.PaymentUI_ErrorValidationUPIWalletAndNB">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="@AfterMethod afterMethod" time="0.001" classname="paymentsUI.PaymentUI_Progress">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.remote.Augmenter.extractRemoteWebDriver(Augmenter.java:72)
at org.openqa.selenium.remote.BaseAugmenter.augment(BaseAugmenter.java:96)
at common.CommonUtil.screenshot(CommonUtil.java:179)
at common.CommonUtil.afterMethod(CommonUtil.java:142)
at paymentsUI.PaymentUI_Progress.afterMethod(PaymentUI_Progress.java:82)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="paymentsUI.PaymentUI_Progress">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="successMessage" time="0.0" classname="paymentsUI.PaymentUI_SuccessfulPaymentMessage">
    <skipped/>
  </testcase> <!-- successMessage -->
  <testcase name="addSCandVerifyIfAdded" time="1.526" classname="paymentsUI.PaymentUI_AddStoredCardAndValidate">
    <skipped/>
  </testcase> <!-- addSCandVerifyIfAdded -->
  <testcase name="errorValidations" time="1.975" classname="paymentsUI.PaymentUI_ErrorValidationUPIWalletAndNB">
    <skipped/>
  </testcase> <!-- errorValidations -->
  <testcase name="addSCandVerifyIfAdded" time="0.875" classname="paymentsUI.PaymentUI_AddStoredCardAndValidate">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:92)
at org.junit.Assert.assertTrue(Assert.java:43)
at org.junit.Assert.assertTrue(Assert.java:54)
at paymentsUI.PaymentUI_AddStoredCardAndValidate.addSCandVerifyIfAdded(PaymentUI_AddStoredCardAndValidate.java:89)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- addSCandVerifyIfAdded -->
  <testcase name="errorValidations" time="0.685" classname="paymentsUI.PaymentUI_ErrorValidationUPIWalletAndNB">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:92)
at org.junit.Assert.assertTrue(Assert.java:43)
at org.junit.Assert.assertTrue(Assert.java:54)
at paymentsUI.PaymentUI_ErrorValidationUPIWalletAndNB.errorValidations(PaymentUI_ErrorValidationUPIWalletAndNB.java:50)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- errorValidations -->
  <testcase name="firePaymentURLandValidatePaymentModes" time="0.806" classname="paymentsUI.PaymentUI_Progress">
    <skipped/>
  </testcase> <!-- firePaymentURLandValidatePaymentModes -->
  <testcase name="firePaymentURLandValidatePaymentModes" time="0.976" classname="paymentsUI.PaymentUI_Progress">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:92)
at org.junit.Assert.assertTrue(Assert.java:43)
at org.junit.Assert.assertTrue(Assert.java:54)
at paymentsUI.PaymentUI_Progress.firePaymentURLandValidatePaymentModes(PaymentUI_Progress.java:58)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- firePaymentURLandValidatePaymentModes -->
  <testcase name="inProgressPayment" time="0.0" classname="paymentsUI.PaymentUI_Progress">
    <skipped/>
  </testcase> <!-- inProgressPayment -->
</testsuite> <!-- Payment UI Validation -->
